src/dstack/_internal/server/utils/logging.py

```diff
@@ -18,25 +18,30 @@ def filter(self, record: logging.LogRecord) -> bool:
 
 
 def configure_logging():
-    default_formatter = "standard"
     formatters = {
-        default_formatter: logging.Formatter(
+        "rich": logging.Formatter(fmt="%(message)s", datefmt="[%X]"),
+        "standard": logging.Formatter(
             fmt="%(levelname)s %(asctime)s.%(msecs)03d %(name)s %(message)s",
             datefmt="%Y-%m-%dT%H:%M:%S",
         ),
         "json": jsonlogger.JsonFormatter(
             "%(asctime)s %(name)s %(levelname)s %(message)s",
             json_ensure_ascii=False,
-            rename_fields={"name": "logger", "asctime": "timestamp"},
+            rename_fields={"name": "logger", "asctime": "timestamp", "levelname": "level"},
         ),
     }
-    log_format = os.getenv("DSTACK_SERVER_LOG_FORMAT", default_formatter)
-    formatter = formatters.get(log_format, formatters[default_formatter])
-
-    root_logger = logging.getLogger(None)
-    handler = logging.StreamHandler(stream=sys.stdout)
-    handler.addFilter(AsyncioCancelledErrorFilter())
+    handlers = {
+        "rich": DstackRichHandler(console=console),
+        "standard": logging.StreamHandler(stream=sys.stdout),
+        "json": logging.StreamHandler(stream=sys.stdout),
+    }
+    if settings.LOG_FORMAT not in formatters:
+        raise ValueError(f"Invalid settings.LOG_FORMAT: {settings.LOG_FORMAT}")
```

Comment:
(nit) Maybe this would be more clear to the user, because the user sets the environment variable, not the variable in `settings`
```suggestion
        raise ValueError(f"Invalid DSTACK_SERVER_LOG_FORMAT: {settings.LOG_FORMAT}")
```

Reply:
True, then maybe fix this and similar in a separate change later!